packages/vite/src/node/server/index.ts

```diff
@@ -486,6 +487,9 @@ export async function createServer(
     res.end('pong')
   })
 
+  //decode request url
```

Comment:
It would be better to add a space🤣
```suggestion
  // decode request url
```